content/en/cloud/get-started-teams/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "How do I get started using the Team feature?"
+type: docs
+tags:
+- team
+- billing
+- SSH
+---
+
+## Create a team
+
+1. In the dashboard, click **Team** at the bottom-left of the dashboard. Then,
+   click **Invite** at the top-right of the Team dashboard.
+
+1. Enter the email address of the person you want to invite to your team.
+   Select their role in the team, either an **Admin** or a **Member**. Then,
+   click **Send invitation**.
+
+   {{% alert title="Warning" color="warning" %}}
+   **Be sure to invite only trusted persons to your team!**
+
+   Currently, the only differences between the _Admin_ and _Member_ roles are
+   that an _Admin_ can:
+
+   - Invite others to the team.
+   - Delete team accounts.
+   - Modify payment information.
+   - Change the team name.
```

Comment:
@shawnrusaw-lambdalabs Is it accurate that these are the only differences between the _Admin_ and _Member_ roles?

Reply:
- remove members

Reply:
What are your thoughts on the suggested change, below?

Teammates with an _Admin_ role can remove other teammates with an _Admin_ role—not just teammates with a  _Member_ role.

```suggestion
   Currently, the only differences between the _Admin_ and _Member_ roles are
   that an _Admin_ can:

   - Invite others to the team.
   - Remove others from the team.
   - Modify payment information.
   - Change the team name.
```

Reply:
i'm fine with that